app/build.gradle

```diff
@@ -74,6 +74,17 @@ android {
     kotlinOptions {
         jvmTarget = "1.8"
     }
+
+    packagingOptions {
+        // Multiple dependency bring these files in. Exclude them to enable
+        // our test APK to build (has no effect on our AARs)
+        excludes += "/META-INF/AL2.0"
+        excludes += "/META-INF/LGPL2.1"
+    }
+}
+
+hilt {
+    enableTransformForLocalTests = true
```

Comment:
comment on what this is for

Reply:
Ah! Never mind, it's not actually needed. Removed it, thanks